InfluxDB Uploader

oliver opened the pull request:
If this feature is enabled and configured in the new config.ini file, Gatemon results are uploaded to the specified InfluxDB server to collect statistics.

oliver committed:
upload received Gatemon results to InfluxDB

This feature is disabled by default since it must be configured first.

oliver committed:
InfluxUploader.php: explicitly send 0.0 values for failed checks

oliver committed:
InfluxUploader.php: also send gatemon name and provider to InfluxDB

oliver committed:
InfluxUploader.php: escape special characters in InfluxDB tag values

oliver committed:
upload single values in fewer request lines

oliver committed:
remove PHP end tags

ghost commented:
Die config.ini wuerde ich als .example ins Repo tun, da man sonst immer unstaged changes hat.

oliver committed:
rename config.ini to config.ini.example

This avoids having the file show up as changed all the time once it
contains the actual configuration values.

oliver committed:
avoid warning if config.ini doesn't exist

oliver commented:
Guter Hinweis; erledigt und noch ein file_exists() hinzugefügt.
